spring-session/src/main/java/org/springframework/session/data/mongo/MongoHttpSessionConfiguration.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Yes - I've made configuration as well as the converters package scoped, which means if I'd like to open annotation to a different package I've to open also those classes.